index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="app-wrap">
+        <div class="header-tab">
+            <div class="tab-title">
+                Ammar Hamlaoui
+            </div>
+            <div class="menu-button">
+                <img src="src/union.png" alt="Menu icon"/>
+            </div>
+        </div>
+        <div class="header-content">
+            <div class="header-primary-text">
+                Hey There. <br> I'm Ammar
```

Comment:
- [ ]  This is the main title and it should be wrapped inside an `<h1>` tag, same reason as above. Kindle make sure to use semantic HTML as much as you can!